runtime/gc_glue_java/ObjectModelDelegate.hpp

```diff
@@ -378,6 +378,21 @@ class GC_ObjectModelDelegate
 		return size;
 	}
 
+	MMINLINE UDATA
+	getHotFieldOffset(MM_ForwardedHeader *forwardedHeader)
+	{
+		uintptr_t preservedSlot = (uintptr_t)(forwardedHeader->getPreservedSlot());
```

Comment:
it would be slightly more readable if you put class pointer calculation into a separate line (and storing into a local) and you don't need separate line for preserved slot than (would be a part of that same class pointer calculation line).